spec/morandi_spec.rb

```diff
@@ -10,19 +10,36 @@
       expect(File.exist?(out))
     end
 
+    context "with a big image and a bigger cropped area to fill" do
+      it 'should create ouptut' do
+        binding.pry
+        settings = {
+          "crop"=>"0,477,15839,18804",
+          "angle"=>90,
+          "fx"=>"colour",
+          "straighten"=>0.0,
+          "gamma"=>0.98,
+          "redeye"=>[]
+        }
+
+        Morandi.process('sample/100_mb_image.jpg', settings, out = 'sample/out_plain.jpg')
```

Comment:
due to the fact that other examples create files at the path `sample/out_plain.jpg`, it might be a good idea to give a name to the output file specific to this example.

if we're expecting the image to be cropped we might also test that the width and height are in line with that

Reply:
Ah! Very good point! Didn't notice that! :) Thanks 
```suggestion
        Morandi.process('sample/100_mb_image.jpg', settings, out = 'sample/out_100_mb_image.jpg')
```